src/core/api/index.js

```diff
@@ -55,7 +55,7 @@ export function callApi({ endpoint, schema, params = {}, auth = false, state = {
     .then((response) => (schema ? normalize(response, schema) : response));
 }
 
-export function search({ api, page, query }) {
+export function adminSearch({ api, page, query }) {
```

Comment:
I think this could have `auth = false` in the args and admin could call `search({ auth: true, ...oldParams })`.